common/components/Forms/AddServiceForm/index.tsx

```diff
@@ -40,33 +57,36 @@ const AddServiceForm: React.FC<Props> = ({ form, edit, currentService }) => {
   })
 
   useEffect(() => {
-    if (!previousMonth) {
+    if (serviceData.domain && serviceData.street) {
+      if (!previousMonth) {
+        form.setFieldsValue({
+          rentPrice: 0,
+          electricityPrice: 0,
+          waterPrice: 0,
+          waterPriceTotal: 0,
+          garbageCollectorPrice: 0,
+          inflicionPrice: 0,
+        })
+        return
+      }
+
+      const {
+        rentPrice,
+        electricityPrice,
+        waterPrice,
+        waterPriceTotal,
+        garbageCollectorPrice,
+        inflicionPrice,
+      } = previousMonth
       form.setFieldsValue({
-        rentPrice: 0,
-        electricityPrice: 0,
-        waterPrice: 0,
-        waterPriceTotal: 0,
-        garbageCollectorPrice: 0,
-        inflicionPrice: 0,
+        rentPrice,
+        electricityPrice,
+        waterPrice,
+        waterPriceTotal,
+        garbageCollectorPrice,
+        inflicionPrice,
       })
```

Comment:
Let's combine this and previous `useEffect`'s to make it look like:

```
useEffect(() => {
    form.setFieldsValue({
        domain: currentService?.domain?._id,
        rentPrice : currentService?.rentPrice ?? previousMonth?.rentPrice?? 0,
        electricityPrice : currentService?.electricityPrice ?? previousMonth?.electricityPrice ?? 0,
        // ...rest
    })
}, [form, previousMonth, currentService])
```

This approach should represent initial values priority rule: `current` > `previous` > `default` for numeric fields